<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="tervetuloaPage">
     <widget class="QPushButton" name="syotaPin">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>300</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&quot;syötä kortti&quot;</string>
      </property>
     </widget>
     <widget class="QLabel" name="tervetuloaLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>311</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mainPage">
     <widget class="QLabel" name="saldoLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>160</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ffffff;&quot;&gt;Tilisi saldo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="saldoLCD">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="nostoPage">
     <widget class="QStackedWidget" name="nostoPageStackedWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="nostoOkPage">
       <widget class="QPushButton" name="nostaButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Nosta</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="nostoVarmistusPage">
       <widget class="QPushButton" name="kyllaNostaButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Kyllä</string>
        </property>
       </widget>
       <widget class="QPushButton" name="eiNostaButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>50</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ei</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="nostoLabel">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="naytaTiedotPage">
     <widget class="QPushButton" name="vaihdaPin">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Vaihda PIN</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>371</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Nimi:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Osoite:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Puhelinnumero:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Tilinumero:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="nimiLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="osoiteLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="puhnroLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="tilinroLabel">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="vaihdaPinLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>340</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="talletusPage">
     <widget class="QStackedWidget" name="talletusPageStackedWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="talletusOkPage">
       <widget class="QPushButton" name="talletaButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Talleta</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="talletusVarmistusPage">
       <widget class="QPushButton" name="kyllaTalletaButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Kyllä</string>
        </property>
       </widget>
       <widget class="QPushButton" name="eiTalletaButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ei</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="talletusLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tilitapahtumaPage">
     <widget class="QTableView" name="tilitapahtumaView">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>461</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tilisiirtoPage">
     <widget class="QStackedWidget" name="siirtoPageStackedWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>181</width>
        <height>91</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="siirtoOkPage">
       <widget class="QPushButton" name="siirraButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Siirrä</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="siirtoVarmistusPage">
       <widget class="QPushButton" name="eiSiirraButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ei</string>
        </property>
       </widget>
       <widget class="QPushButton" name="kyllaSiirraButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Kyllä</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="siirtoTilinumeroPage">
       <widget class="QPushButton" name="syotaTilinumero">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>111</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Syötä tilinumero</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="siirtoLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>211</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="vikatilaPage">
     <widget class="QLabel" name="vikatilaLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>200</y>
        <width>521</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Pin Väärin. Kortti lukittu.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="summatWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="summa10">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="summa20">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="summa50">
       <property name="text">
        <string>50</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="summa100">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="summa500">
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="summaMuu">
       <property name="text">
        <string>muu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="valikkoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>161</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QPushButton" name="kirjauduUlos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Kirjaudu ulos</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>141</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="etusivu">
        <property name="text">
         <string>Etusivu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nosto">
        <property name="text">
         <string>Nosto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="talletus">
        <property name="text">
         <string>Talletus</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tilisiirto">
        <property name="text">
         <string>Tilisiirto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tilitapahtumat">
        <property name="text">
         <string>Selaa tilitapahtumia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="naytaTiedot">
        <property name="text">
         <string>Näytä tietosi</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="paaOtsikkoLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
